src/api/auth/token.py

```diff
@@ -0,0 +1,27 @@
+import structlog
+from fastapi import Request
+
+from src.core.exceptions import InvalidToken, TokenNotProvided
+from src.depends import Container
+
+log = structlog.get_logger()
+
+
+async def check_header_contains_token(request: Request):
+    """Проверяем, содержится ли в заголовке запроса token, и сравниваем его
+    со значением ACCESS_TOKEN_FOR_PROCAHRITY из settings.py"""
+    settings = Container().settings()
+    if not hasattr(settings, "ACCESS_TOKEN_FOR_PROCHARITY"):
```

Comment:
Запиши этот атрибут в модельке settings, иначе работать не будет.

```suggestion
    if not settings.ACCESS_TOKEN_FOR_PROCHARITY:
```